### PR TITLE
Cluster domain test

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What type of PR is this?**
integration-test

**What this PR does / why we need it**:

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

### CODE REVIEW

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -98,9 +101,11 @@ func TestMain(t *testing.T) {
 	}
 	// we can either run cluster domain tests or the other tests
 	// the underlying setup of clusters is different
-	if os.Getenv("ENABLE_CLUSTER_DOMAIN_TESTS") != "" {
+	if _, err := strconv.ParseBool(os.Getenv(enableClusterDomainTests)); err == nil {
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: You also have to check the value of the bool ie the first return value from `ParseBool`

File: test/integration_test/test-deploy.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@ snapshot_scale=10
 image_name="stork:master"
 test_image_name="stork_test:latest"
 remote_config_path=""
+run_cluster_domain_test=true
```
Comment (disrani-px) [inline test/integration_test/test-deploy.sh]: Set to `false` by default

File: test/integration_test/common_test.go (hunk 1 above)
Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

File: test/integration_test/test-deploy.sh (hunk 2 above)
Comment (Rohit-PX) [reply test/integration_test/test-deploy.sh]: Fixed.

File: test/integration_test/common_test.go [hunk 3]
```diff
@@ -98,9 +101,13 @@ func TestMain(t *testing.T) {
 	}
 	// we can either run cluster domain tests or the other tests
 	// the underlying setup of clusters is different
-	if os.Getenv("ENABLE_CLUSTER_DOMAIN_TESTS") != "" {
-		t.Run("ClusterDomains", testClusterDomains)
+	if enabled, err := strconv.ParseBool(os.Getenv(enableClusterDomainTests)); err == nil {
+		if enabled {
```
Comment (disrani-px) [inline test/integration_test/common_test.go]: If enabled is false this won't run any test

Comment (Rohit-PX) [reply test/integration_test/common_test.go]: Fixed.

**Pull request merged**